api/src/flows.ts

```diff
@@ -421,9 +421,11 @@ class FlowManager {
 
 		const handler = this.operations.get(operation.type)!;
 
-		const options = applyOptionsData(operation.options, keyedData);
+		let options = null;
```

Comment:
Any opinion on if we should default to `null` or `operation.options`? I can see arguments for both.